### PR TITLE
Avoid to always round the range in connectRange

### PR DESCRIPTION
**Pull request by samouss**
**Summary**

In the `connectRange` we always round the range even with a precision greater than 0. Now we round the range only for a precision of 0 and leave the original value in the other case.

**Result**

You can play with the `RangeInput` on dev-novel.

**Before:**

screen shot 2017-10-18 at 11 48 02

**After:**

screen shot 2017-10-18 at 11 40 49

### CODE REVIEW

File: src/connectors/range/connectRange.js [hunk 1]
```diff
@@ -100,8 +100,8 @@ export default function connectRange(renderFn) {
         }
 
         return {
-          min: Math.floor(min),
-          max: Math.ceil(max),
+          min: precision === 0 ? Math.floor(min) : min,
```
Comment (bobylito) [inline src/connectors/range/connectRange.js]: What does it mean for a precision of 1 and there 3 digits after the `.` or `,`?

Comment (samouss) [reply src/connectors/range/connectRange.js]: When I begin to implement it, I thought it was a bit overkill.

Comment (bobylito) [reply src/connectors/range/connectRange.js]: Well Isn't the purpose to bring consistency? And in this case, we only have consistency for precision = 0, right?

Comment (bobylito) [reply src/connectors/range/connectRange.js]: But then I might not have a good idea of the complexity...

Comment (samouss) [reply src/connectors/range/connectRange.js]: Right, it will update it 🙃

**Pull request merged**